gulpfile.js

```diff
@@ -106,9 +106,11 @@ var cssFilePaths = [];
 var jsFilePaths = [];
 var fontFolderPaths = [];
 var cssBackgroundFilepaths = [];
+// This cache slug should be same as feconf.CACHE_SLUG_DEV
```

Comment:
Nothing substantial. Looks cleaner. Easier if we decide to add a dev slug in the future.